Heroes Handbook, Main File.txt

```diff
@@ -837,76 +837,95 @@ As a lightforged draenei, you've commited yourself to a crusade against the burn
 
-***Size.*** Humanly varying widely in height and build, from barely 5 feet to well over 6 feet tall. In worgen form, they range from 7 to 9 feet. Your size is Medium.
+***Size.*** Like humans, they vary widely in height and build, most standing well over 6 feet tall, and weigh around 250 pounds. Your size is Medium.
 
-***Speed.*** Your base walking speed is 30 feet.
+***Speed.*** Your base walking speed is 30 feet
 
-***Beastly Appearance.*** You are proficient in the Intimi-dation skill, and add twice your proficiency bonus to it.
+***Darkvision.*** Accustomed to the black nights, you have superior vision in dark and dim conditions. You can see in dim light within 60 feet of you as if it were bright light, and in darkness as if it were dim light. You can't discern color in darkness, only shades of gray.
 
-***Superior Darkvision.*** Accustomed to the black nights, you have superior vision in dark and dim conditions. You can see in dim light within 120 feet of you as if it were bright light, and in darkness as if it were dim light. You can't discern color in darkness, only shades of gray.
+***Hunter's Lore.*** You gain proficiency with one of the following skills of your choice: Animal Handling, Nature, Perception, Stealth, or Survival.
 
-***Language of Wolves.*** You are able to communicate with wolves, as the spell *commune with animals*.
+***Languages.*** You can speak, read, and write Common and one extra language of your choice. Humans typically learn the languages of other peoples they deal with, including obscure dialects.
 
-***Natural Weapons.*** Your claws and bite are natural weapons, which you can use to make unarmed strikes. If you hit with your claws, you deal 1d4 + your Strength modifier slashing damage, and with your bite, you deal 1d6 + your Strength modifier piercing damage.
+***Two Forms.*** Using 1 minute, you can transform yourself into a human. While human, you appear as you did before getting affected by the worgen curse. You always appear as the same human when you transform. You can transform back into a worgen as a free action on your turn.
 
-***Truesilver Vulnerability.*** You are vulnerable to damage taken by silvered weapons.
+As a human, you loose all worgen form traits. If you make an attack, cast a spell that affects an enemy creature, or try to use an ability only available in worgen form, you imme-diately revert into your worgen form.
 
-***Viciousness.*** When you score a critical hit with a melee attack, you can roll one of the weapon's damage dice one additional time and add it to the extra damage of the critical hit.
+<div style="text-align:Center"> 
 
-***Languages.*** You can speak, read, and write Common and one extra language of your choice. Humans typically learn the languages of other peoples they deal with, including obscure dialects.
+#### Worgen Form Traits
+</div>
+
+&nbsp;&nbsp;&nbsp; ***Speed.*** You base walking speed is 35 ftt, and you have a climbing speed of 20 feet.
```

Comment:
```suggestion
&nbsp;&nbsp;&nbsp; ***Speed.*** You base walking speed is 35 feet, and you have a climbing speed of 20 feet.
```

Are we normalizing distance numbers as 'feet' or 'ft.'?

Reply:
Usually it's 'feet' for this kind of thing, must've just slipped my mind.